ztp/source-crs/optional-extra-manifest/ipsec/ipsec-endpoint-config.yml

```diff
@@ -0,0 +1,15 @@
+interfaces:
```

Comment:
is this for day2 config? If so, it should stay in a different folder than day0 config so that for day0, user can include a folder instead of individual files.

Reply:
nit: should be .yaml to be consistent in the project.

Reply:
@sabinaaledort a day-2 config can't stay in day-0 manifests folder as Yang mentioned.

Reply:
As a counterpoint, this is not enabled by default, so in my opinion the question of *where* it lives inside the ZTP container is less important compared to the facts that:
- It is not enabled by default
- We provide good instructions to customers on how to enable it

There are other places it could go; one would be under `ztp/gitops-subscriptions/argocd/examples/`.  But keep in mind that the `source-crs/` hierarchy already has a combination of day-0, day-2, and optional content that can be enabled at either phase by a customer opt-in adding the yamls to the right extra-manifests or PGT as needed.